adapters/pom.xml

```diff
@@ -113,6 +113,10 @@
       <groupId>org.eclipse.hono</groupId>
       <artifactId>service-base-test-utils</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-service-base</artifactId>
+    </dependency>
```

Comment:
Corresponding dependency can be removed from the dependent "hono-adapter-base-spring" module then.